### DISCUSSION TITLE
JSON datatype marked as obsolete in documentation, will it be removed?

### DISCUSSION THREAD

**Discussion by ikss** [ClickHouse/ClickHouse#62841] [General]
In [JSON type doc page](https://clickhouse.com/docs/en/sql-reference/data-types/json) stated that JSON type is obsolete and shouldn't be used.
So I have a question, will it be removed or replaced in some future?

**Comment by Algunenano** [comment]
Yes, it will be removed.

It will be replaced by https://github.com/ClickHouse/ClickHouse/issues/54864 (already partially implemented)

**Comment by ikss** [comment]
Thank you Raúl for prompt reply
